core/src/main/java/org/infinispan/configuration/cache/IndexingConfiguration.java

```diff
@@ -84,11 +87,17 @@ public TypedProperties properties() {
 
    /**
     * Returns the indexing mode of this cache.
+    * @deprecated This configuration can be removed as the index mode is calculated automatically.
     */
+   @Deprecated
    public Index index() {
       return index.get();
    }
 
+   public boolean isEnabled() {
```

Comment:
It seems that tradition is to just call it ```enabled```. The majority of our boolean options in config do not start with "is"